lib/acm_bot/bot.ex

```diff
@@ -10,10 +10,20 @@ defmodule AcmBot.Bot do
 
   def bot(), do: @bot
 
-  def handle({:command, "start", _message}, context) do
+  command("start")
+  command("armario")
+
+  def handle({:command, :start, _message}, context) do
     answer(context, "Hola buenas tardes!\nEste es el nuevo bot de ACM.")
   end
 
+  # Versión precaria del comando armario. Es muy mejorable con las ideas de @IronJanoWar
+  def handle({:command, :armario, msg}, context) do
+    chat_id = msg.chat.id
+    prices_path = ExGram.Config.get(:acm, :prices_file)
```

Comment:
Right now this could fail, producing unexpected behaviors like the bot not answering to a `/armario` command.
I suggest using a `case` clause to check if `prices_path` is `nil` and send an error message to the Telegram user if that's the case.
Take into account that `ExGram.Config.get(:acm, :non_existing_key)` returns `nil` by default if the `:non_existing_key` is not defined in the config.

Reply:
Why using a `case` instead `if`. I don't know, but I see useful using `if` here.

Reply:
`if` is just a macro that transform to a `case`, example:

```elixir
if is_nil(x) do
   x
else
  x
end
```

In reality is:

```elixir
case is_nil(x) do
  false -> x
  true -> x
end
```

And, the bytecode between that 2 options and a case like this is the same:

``` elixir
case x do
   nil -> x
   _ -> x
end
```

(Yes, that 3 code's bytecode are the same)

So, that being said, I usually avoid `ifs` because I think that matching for the actual value is more explicit and you can see much better what do you want to accomplish.

Reply:
Ooooooh. So cool. Thanks <3